pages/docs/how-to-guides/events.mdx

```diff
@@ -73,15 +76,12 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
 | country           | false    | In which country, the event is hosting |
 
 4. Now you can commit your file and create a Pull Request. For more details please see <Link href="/docs/how-to-guides/editing">Editing Guide</Link>
-
-If you want to know how to properly add start, end, and CFP Close times, feel free to check [Unix Time stamp](https://www.unixtimestamp.com/).
-
-export default ({ children }) => (
-  <DocsLayout
+   If you want to know how to properly add start, end, and CFP Close times, feel free to check [Unix Time stamp](https://www.unixtimestamp.com/).
```

Comment:
Add a space here and fix the formatting with prettier

Reply:
I dont think you need to change the config just install prettier in vscode then right click, format with prettier

Reply:
Thanks Eddie and Sara for fixing the issue. It was my  first time for me  fixing the doc, Thanks to both of you and all the  folks who guided me.